### DISCUSSION TITLE
Are there any plans to allow user accounts that are not in any organizations use CodeSpaces?

### DISCUSSION THREAD

**Discussion by behrangsa** [community/community#34062] [Codespaces] [answered]
Hi,

If GitPod can do that, surely GitHub can do that too.

I have dozens of personal private projects under my user account and I don't want to move them to an organization to use them in CodeSpaces. GitPod can do it so surely GitHub should be capable of doing that too.

So the question is: are there any plans to let users open their personal repositories in CodeSpaces?

Thanks.

**Comment by salvador-barboza** [comment, marked as answer]
Hello @behrangsa! We are working to make Codespaces available for all users later this year.
